extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -13,3 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+amp-sidebar {
+  color: black;
+  background-color: #efefef;
+}
+
+amp-sidebar::part(sidebar) {
+  color: inherit;
+  background-color: inherit;
```

Comment:
Should we go ahead and immediately propagate other CSS props here, such as `min/maxWidth/width`, `min/maxHeight/height`, `border`, and `padding`. Not sure if any other apply off the top of my head, but possible.

Reply:
Yep, I have added more CSS properties that can be overwritten by the user in this selector.  Example usage is the in the amp storybook.  Let me know what you think.

I will follow up with tests and Preact storybook if it looks okay to you.  Thanks!